pkg/client/unversioned/request.go

```diff
@@ -98,6 +98,13 @@ type Request struct {
 
 	apiVersion string
 
+	// group/version
+	group      string
+	version    string
```

Comment:
How is this field different from `apiVersion` above?  Does this change the way that `baseURL` is used?  

Also, I remember an `UnversionedPath` method that did some crazy things.  This probably has effects tehre too.